docs/reference/monitoring/local-export.asciidoc

```diff
@@ -93,3 +93,13 @@ xpack.monitoring.history.duration: 3d <2>
 <2> Lower the default history duration from `7d` to `3d`. The minimum value is
 `1d`. This setting can be modified only when using a Gold or higher level
 license. For the Basic license level, it uses the default of 7 days.
+
+To disable cleaner service on the monitoring cluster, a disabled local exporter can be added :
+[source,yaml]
+---------------------------------------------------
+xpack.monitoring.collection.enabled: false
```

Comment:
The following is necessary to disable the cleaner: (setting this to false would also disable all monitoring data collection)
```
xpack.monitoring.collection.enabled: false
```